### DISCUSSION TITLE
Blender Octane's Export Settings

### DISCUSSION THREAD

**Discussion by Meloq-Arai** [sobotka/AgX#7] [Q&A] [answered]
Hello!

First of all, I must say that I'm by no means an expert in this topic, however, this development seems very exciting and I would like to integrate it into my workflow, but I have some questions that I can't figure out on my own.

I'm using Blender with Octane plugin, and recently decided to try out AgX. However, I can't quite get the same results in the viewport and in EXR export.

I set Octane's Color Management to Automatic

To view I'm using Octane's Camera Imager's OCIO settings set to 
OCIO view: sRGB: AgX
OCIO look: Punchy

Blender's color management is:
Display Device: sRGB
View Transform: Raw
Look: None

And for export I've tried
Color Space: AgX Base (Imagery), OR I've tried Appearance Punchy sRGB
OCIO look: Punchy

To view the EXR I've used photoshop, and it just looks wrong inside of it. (I don't know, maybe that's the problem)
I've also attached a screenshot that shows viewport image - left, and oversaturated colors in photoshop - right.

I'd like to know what am I missing here. Thanks.

![photo_2022-11-18_15-42-41](https://user-images.githubusercontent.com/118545633/202731071-e80eb4c5-d161-49fc-84c1-5dd9f1d27c37.jpg)

**Comment by sobotka** [comment, marked as answer]
> Color Space: AgX Base (Imagery), OR I've tried Appearance Punchy sRGB

The version in this repository has no looks for technical reasons and compatibility.

The base image formed will be slightly less chroma, and generally flatter value distribution.

If you test with only the View, you should see 1:1. At the very least, it gives us a more useful ground truth to build on.

> I can't quite get the same results in the viewport and in EXR export.

Quite frequently the EXR encoding is best considered a not-yet-formed picture. Thinking of the tristimulus values as data, akin to normals or depth, can help here.

This typically means that the encoding in the rendered tristimulus does not match what the formed image should be, and requires that the tristimulus render data be rolled through the image formation chain in order to see a picture.

Photoshop would then load the EXR as tristimulus and apply the vendor specific picture formation, which in many cases, is do nothing and defer the distortions to whatever the display does. It would be wisest to output a higher bit depth TIFF, unless Blender can bake the encoding of the picture into the EXR. Last I checked, this was not quite possible, but potentially.

First step to sorting things out would be to get the ground truths firm, and then solving the problems should be easier.

Also note that the configuration in this repository requires OpenColorIO version 2.0 or above.

**Reply by Meloq-Arai** [reply]
Thank you for the explanation, I still have a lot to learn and this is very interesting and useful!
